modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -99,9 +100,9 @@ Returning to the Good Weather use case, imagine that the company uses the follow
 
 image::multiple-IdP-user-flow.png[User Flow for Configuring Multiple External Client Providers]
 
-As shown in the diagram, you first configure your external client provider and assign it to the appropriate business groups and environments. When you create or modify an API within a specific business group or environment, the new client provider is displayed. You can reselect the appropriate client provider. 
+As shown in the diagram, you first configure your external client provider and assign it to the appropriate environments. When you create or modify an API within a specific environment, the new client provider is displayed. You can reselect the appropriate client provider. 
```

Comment:
Clarify or simplify: you first configure and assign, but if you add or modify an API in an environment, you [might?] need to change to a new provider?